.github/workflows/ci.yml

```diff
@@ -22,10 +22,10 @@ jobs:
             ruby_version: "2.7"
           - label: Ruby 3.0
             ruby_version: "3.0"
-          - label: Ruby 3.1
-            ruby_version: "3.1"
-          - label: JRuby 9.3.4.0
-            ruby_version: "jruby-9.3.4.0"
+          - label: Ruby 3.1.2
+            ruby_version: "3.1.2"
```

Comment:
What's our strategy/timeline for opening this back up to 3.1.x? Definitely don't want to fall behind...

Reply:
The plan is to manually monitor developments in the ruby_builder project repo and test on my personal fork of the Jekyll repo.
Update workflow here once the tests run successfully.